docs/gameplay-sessions/gameplay-sessions-reference.mdx

```diff
@@ -222,4 +222,38 @@ Any time one of these events occurs on a tick, the corresponding key frame will
 Performance information, such as fps, render time, and object stats are either calculated frame-by-frame, or use
 the [UnityStats](https://stephenhodgson.github.io/UnityCsReference/api/UnityEditor.UnityStats.html) class. Since
 UnityStats is an editor-only feature, some performance information, specifically that of `engineStats`, is only
-available when running in the Unity Editor.
\ No newline at end of file
+available when running in the Unity Editor.
+
+## RGTestUtils class
+
+The `RGTestUtils` class is a utility class that provides methods for loading and playing back recordings in Unity
+Play Mode tests. This class is included in the SDK and can be used to automate the testing of your game.
+
+### API
+
+```csharp
+namespace RegressionGames
```

Comment:
Non-blocking I guess, but a re-occurring discussion point in doc reviews

We really shouldn't be documenting APIs manually on the doc site.

If we want to create an automatically generated API docs site from the build that sounds fine/good, but if I update the code, I highly doubt I'll remember to come retype this doc page.